dev/com.ibm.ws.beanvalidation.core/resources/com/ibm/ws/beanvalidation/resources/nls/BVNLSMessages.nlsprops

```diff
@@ -87,3 +87,16 @@ JNDI_NON_JEE_THREAD_CWNBV0006E.explanation=JNDI operations on a java:comp/env na
 JNDI_NON_JEE_THREAD_CWNBV0006E.useraction=Modify the application so that the JNDI operation occurs on a thread associated with a Java EE application component.
 
 # -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the method with the invalid ValidateOnExecution annotation
+# -----------------------------------------------------------------------
+BVKEY_VALIDATE_ON_EXECUTION_NOT_ALLOWED_CWNBV0007E=CWNBV0007E: @ValidateOnExecution is not allowed on methods overriding a superclass method or implementing an interface. Check configuration for {0}
```

Comment:
change
on methods overriding
to
on methods that override

Reply:
Your requested updates have been made for this comment and the others in a new commit.